frontend/packages/kubevirt-plugin/src/k8s/patches/vm/vm-patches.ts

```diff
@@ -1,6 +1,158 @@
-import { Patch } from '@console/internal/module/k8s';
-import { VMLikeEntityKind } from '../../../types';
+import * as _ from 'lodash';
+import { Patch, TemplateKind } from '@console/internal/module/k8s';
+import { VMLikeEntityKind, VMKind, CPU } from '../../../types';
 import { getAnnotations, getDescription } from '../../../selectors/selectors';
+import { getFlavor, getCPU, getMemory, isVM } from '../../../selectors/vm';
+import { CUSTOM_FLAVOR, TEMPLATE_FLAVOR_LABEL } from '../../../constants';
+import { selectVM, getTemplateForFlavor } from '../../../selectors/vm-template/selectors';
+import { getVMLikePatches } from '../vm-template';
+import { isCPUEqual } from '../../../utils';
+
+const getLabelsPatch = (vmLike: VMLikeEntityKind): Patch | null => {
```

Comment:
```suggestion
const getLabelsPatch = (vmLike: VMLikeEntityKind): Patch => {
```

we dont have strict null check enabled, so no need to add `| null`. Applies for others too

Reply:
leftover, removed